src/pages/Profile.vue

```diff
@@ -80,7 +80,6 @@
 
         <q-btn
           rounded
-          color="secondary"
           label="Speichern"
```

Comment:
Could you do i18n for that?

Reply:
Yes. Let's do this in another PR because to keep the purpose of this one clear.